packages/kit/src/core/sync/sync.js

```diff
@@ -27,6 +28,7 @@ export async function create(config) {
 	const output = path.join(config.kit.outDir, 'generated');
 
 	write_client_manifest(config, manifest_data, output);
+	write_server(config, output);
```

Comment:
So this means that we made the generated code a little smaller and it's the same mechanism between dev and prod?

Reply:
exactly, yeah — less duplication, and less per-request work in dev